ci-operator/step-registry/netobserv-cli/tests/netobserv-cli-tests-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+
+set -x
+short_sha=$(git rev-parse --short HEAD)
+USER=netobserv VERSION=$short_sha make commands
+export PATH=$PATH:$PWD/build
+GINKGO_VERSION=$(go list -mod=readonly -m -f '{{ .Version }}'  github.com/onsi/ginkgo/v2)
+export GINKGO_VERSION
+# download the ginkgo cli binary
+go mod tidy
```

Comment:
```suggestion
```

Why is that needed here ? I don't think it's the case

Reply:
I guess it's not, since we already have vendor directory, let's remove it try it.